docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 version: '3.0'
 services:
   orca:
-    image: pcic/orca:latest
+    image: pcic/orca:i23-create-production-app
```

Comment:
Did you mean to leave `i23-create-production-app`?

Reply:
I definitely did not.